app/src/main/res/values/strings.xml

```diff
@@ -6,6 +6,7 @@
     <string name="cron_input_hint">Cron expression (Quartz)</string>
     <string name="notification_title_input_hint">Notification title</string>
     <string name="notification_text_input_hint">Notification text</string>
+    <string name="on_click_uri_input_hint">On click open uri</string>
```

Comment:
I would prefer "URI (opens on click)", "URI (öffnet sich bei Klick)", "URI (opent op klik)"
The German and Dutch translation is missing

Reply:
I've made the changes and added the translation, though you should probably give it a look as I don't speak either of those languages.